src/Playwright/Transport/ChannelOwnerBase.cs

```diff
@@ -136,4 +136,15 @@ private void UpdateEventSubscription(string eventName, bool enabled)
             }),
             true).IgnoreException();
     }
+
+    internal Task<JsonElement?> SendMessageToServerAsync(
```

Comment:
why do we need this specialization of the template?

Reply:
Some legacy, let me try to follow-up with that.

Reply:
Turns out we have it twice:

a) SendMessageToServerAsync without the generic gives you a JsonElement? back.
b) SendMessageToServerAsync with a generic which internally deserialises the JSON for you into some specific data structure T

So no need to follow-up!